WIP: upgrade instructions (#23)
elastic/prodfiler-documentation: ecs.md

## Deploying Prodfiler 

> Both scripts are _not_ idempotent, so they should be executed from AWS credentials 

Reviewer: from AWS credentials ?

Author: Thanks, fixed 🙏🏼

To see the help message, execute the scripts with `-h` or `--help`.

### Quick start

Follow the instructions in the `... > Instructions` menu from the Prodfiler UI, switching to the ECS tab.
You will be guided through the execution of the scripts mentioned above.

### Customize the setup

> If you need to adjust the default settings, follow along to know more about how the host-agent installation works.

Reviewer: To put it simpler: (just a suggestion)
"The following describes how to change the default settings." ?

Author: Updated 🙏🏼

#### Authentication and secrets

Pass the provided credentials as arguments to this script in order to access the private Docker image.

> Replace the `<PASSWORD>` placeholder content with the token you can see in the `Instructions` menu.

Reviewer: ```suggestion
Replace the `<PASSWORD>` placeholder content with the token shown in the `Instructions` menu.
```

It will create the secrets used by the ECS DAEMON task using AWS Secret Manager.

#### ECS task

Fetch the `projectID` and `secretToken` values visible in the Prodfiler web UI
when creating a new project.
The values are defined as environment variables in the deployment command visible in the UI,
they are called `PRODFILER_PROJECT_ID` and `PRODFILER_SECRET_TOKEN` respectively.

> To customize the setup, change the `task.json.template` that you can find [here](scripts/ecs/awscli/task.json.template),
> adding or changing the configurations required by your environment.

Reviewer: ```suggestion
To customize the setup, edit [task.json.template](scripts/ecs/awscli/task.json.template) as required by your environment.
```
